CONTRIBUTING.md

```diff
@@ -29,6 +29,50 @@ submitted for inclusion in the work by you, as defined in the
 Apache-2.0 license, shall be dual licensed as in the README, without any
 additional terms or conditions.
 
+## Version numbers
+
+We use `git-testament` to construct our version strings. This includes into
```

Comment:
```suggestion
We use `git-testament` to construct our version strings. This includes
```

Reply:
This is a misunderstanding of a badly worded phrase "This includes-into-the-binary ..." I shall attempt to reword more nicely.

Reply:
```markdown
We use `git-testament` to construct our version strings. This records, as a
struct, details of the git commit, tag description, and also an indication
of modifications to the working tree present when the binary was compiled.
```